src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,13 @@
+import "./ColorButton.scss";
+
+const ColorButton = (props) => {
+  const switchColor = () => {
+    props.setColor(props.color);
+  };
+  return (
+    <>
+      <button className={`colorButton ${props.color}`} onClick={switchColor} />
+    </>
```

Comment:
```suggestion
      <button className={`colorButton ${props.color}`} onClick={switchColor} />
```
하나의 태그로 되어있어서 fragment 태그 사용하지 않으셔도 됩니다!